.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out branch
+        uses: actions/checkout@v3
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "6.x.x"
+
+      - name: Generate Certificates
+        run: dotnet dev-certs https --export-path ./cert/cert.crt --no-password --format PEM
+
+      - name: Install Modules
+        run: |
+          npm i ./framework/build/sitefinity-react-framework-0.1.0.tgz
```

Comment:
Is it possible to publish this to a npm feed? Keeping it in the source doesn't feel right.

Reply:
No need for feeds, with new structure we do not have tgz files in repo.